x-pack/filebeat/module/azure/activitylogs/ingest/pipeline.yml

```diff
@@ -39,6 +39,7 @@ processors:
 - grok:
     field: azure.activitylogs.callerIpAddress
     patterns:
+      - "%{IPV6:source.ip}"
```

Comment:
One last question: wouldn't using `{IP:source.ip}` instead of `{IPV6:source.ip}` yield the same result?